web/source/osk/visualKeyboard.ts

```diff
@@ -60,13 +65,12 @@ namespace com.keyman.osk {
     currentTarget: KeyElement;
 
     // Popup key management
-    popupBaseKey: KeyElement;
-    popupPending: boolean = false;
-    subkeyDelayTimer: number;
     popupDelay: number = 500;
+    subkeyDelayTimer: number;
     menuEvent: KeyElement; // Used by embedded-mode.
     keytip: KeyTip;
     subkeyPopup: browser.SubkeyPopup;
+    subkeyDelegator: any; // a temp, intermediate property during subkey abstraction work
```

Comment:
Extra comment for reviewers:  this is an 'embedded' mode parallel to the `browser.SubkeyPopup` class.

As `SubkeyPopup` manages in-browser ('native-mode') subkey selection, the "subkey delegator" introduced here is used to "manage" (integrate) embedded-mode subkey selection.

Reply:
This introduces the parallel now; later PRs in the chain will bring them closer together and lead to a point where an abstract interface for both may be defined.